.golangci.yml

```diff
@@ -36,7 +36,6 @@ linters-settings:
 
 linters:
   enable:
-    - revive
```

Comment:
I'm removing `revive` since, for some reason, it was firing that `moduleCloser`, which is `func(context.Context) error`, is a _duplicate_ of an existing type (e.g: `io.Close`). 

OK, that's true, but I won't use `io.Closer`. In any case, I'm not sure I've ever seen any useful lint from `revive` anyway.